src/sitemap-builder.mjs

```diff
@@ -0,0 +1,12 @@
+import Sitemap from 'react-router-sitemap';
+import fs from 'fs';
+const jsonString = fs.readFileSync('/home/panspagetka/insights/insights-rbac-ui/package.json');
```

Comment:
This will not work, can you change it to `import { routes } from './package.json'`?

Reply:
Thanks for the catch ;)  Node doesn't like `import { routes } from './package.json'` but `import packageJson from './package.json'` is just fine.